src/data_loader/population/populationGeoDataFrame.py

```diff
@@ -0,0 +1,106 @@
+## This file is for fetching the population data from datapop and outputting af GeoPandas dataframe with population snapped to nodes.
+import logging
+
+import geopandas as gpd
+import networkx as nx
+import numpy as np
+import osmnx as ox
+import rasterio.warp
+from shapely.geometry import Point
+from tqdm import tqdm
+
+tif_path = "../../../data/population/"
+tif_filename = "dnk_ppp_2020_constrained.tif"
+tif_filenamePath = f"{tif_path}{tif_filename}"
+logging.getLogger().setLevel(logging.INFO)
+
+
+def download_osm_graph(queries: list[str]) -> nx.MultiDiGraph:
+    def download_query(query: str) -> nx.MultiDiGraph:
+        logging.info(f"Downloading graph for {query}")
+        city_graph = ox.graph_from_place(
+            query, network_type="drive_service", simplify=True
+        )
+        logging.info(f"Downloaded graph for {query}")
+        return city_graph
+
+    return nx.compose_all([download_query(city) for city in queries])
+
```

Comment:
Same as before, just import the function
```suggestion
from data_loader.osm import download_osm_graph

tif_path = "../../../data/population/"
tif_filename = "dnk_ppp_2020_constrained.tif"
tif_filenamePath = f"{tif_path}{tif_filename}"
logging.getLogger().setLevel(logging.INFO)
```